### ISSUE TITLE
ITypeDefinition get additionalProperties value

### ISSUE THREAD

**Issue by domarmstrong** [raml-org/raml-js-parser-2#611]
```
types:
  Foo:
    type: object
    additionalProperties: false
    properties:
      foo: string
```

For example I have Foo as a runtimeType, but cant seem to get the value of additionalProperties.

```
type.facets() -> []
type.allFacets() -> []
type.allFixedFacets() -> {}
```

Any suggestions?

**Comment by KonstantinSviridov** [comment]
Hi, @domarmstrong 

It's the same story as with the `default` facet: https://github.com/raml-org/raml-js-parser-2/issues/600
`additionalProperties` value is going to be available in 1.1.14 by means of `fixedBuiltInFacets` and `allFixedBuiltInFacets` methods.

Regards, Konstantin

**Comment by domarmstrong** [comment]
Thanks. I tested from develop already ^_^
